tests/unit/aiplatform/test_featurestores.py

```diff
@@ -1054,6 +1123,50 @@ def test_ingest_from_gcs_with_invalid_gcs_source_type(self):
                 gcs_source_type=_TEST_GCS_SOURCE_TYPE_INVALID,
             )
 
+    @pytest.mark.usefixtures("get_entity_type_mock")
+    def test_read_single_entity(self, read_feature_values_mock):
```

Comment:
Can we have a test which the response is parsed, i.e have multiple columns/features, some of them are empty?

Reply:
added test case for all value type, with missing data value scenario